### ISSUE TITLE
about total loss

### ISSUE THREAD

**Issue by www516717402** [ultralytics/yolov5#3255]
Thank you for open this repo.
https://github.com/ultralytics/yolov5/blob/7b863a34dcacf8b310396d0bc1021f404ec85797/utils/loss.py#L161

Why multiply batchsize in total_loss ?

**Comment by glenn-jocher** [comment]
@www516717402 without this line your training results will change as your batch size changes.

**Comment by www516717402** [comment]
@glenn-jocher https://github.com/ultralytics/yolov5/blob/7b36e38cf8f3d3c08e973b18913ae8e41ff970b2/utils/loss.py#L142 BCELoss already devide batchsize, so loss is a mean number and it could no-relate with batchsize. The result is terrible when remove multiply batchsize. Its such big confuse for me.

**Comment by glenn-jocher** [comment]
@www516717402 theres nothing confusing. The loss is multiplied by batch size on L161 to allow training at any batch size. See batch-size study for results: https://github.com/ultralytics/yolov5/discussions/2452

I would not modify the code if I were you unless you are a PyTorch/YOLO expert.
